dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AlertWarnLevel.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.enums;
+
+public enum AlertWarnLevel {
+
+    MIDDLE,SERIOUS
```

Comment:
Is it better to match the logger?
For excample: debug, info, warning, error

Reply:
> Is it better to match the logger?
> For excample: debug, info, warning, error
This is a way of storing the string directly into the content of the message and sending it out, and if it's a number it needs to be sent out under conversion.

Reply:
![image](https://user-images.githubusercontent.com/59079269/90854112-ab8a3e80-e3ae-11ea-8c9e-1adadf86d3ee.png)
![image](https://user-images.githubusercontent.com/59079269/90854123-b644d380-e3ae-11ea-9185-cee15f098059.png)